src/main/java/org/zalando/nakadi/util/TimeLogger.java

```diff
@@ -0,0 +1,130 @@
+package org.zalando.nakadi.util;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+
+public class TimeLogger {
```

Comment:
I like this class, but I think that we can avoid doing that just by creating time logger within the method (in local method scope)